test/functional-tests/test.sh

```diff
@@ -188,6 +188,29 @@ function progress_bar {
     keypress
 }
 
+function icon_position {
+    for position in left top right off; do
+        tmp_dunstrc dunstrc.icon_position "icon_position = $position"    # use default icon theme
+        start_dunst dunstrc.tmp
+        for urgency in l n c; do
+            ../../dunstify -a "dunst tester" -u $urgency "icon_position = $position"
+        done
```

Comment:
Maybe add a few tests for testing padding in combination with icon position. So these same tests, but then with all the paddings set to different values.